src/agentscope/utils/mixture_of_agent.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
```

Comment:
Plz create a directory `strategy` under `src/agentscope` for the file `mix_of_agents.py`.

Reply:
Done